crates/rbuilder/Cargo.toml

```diff
@@ -7,6 +7,8 @@ build = "build.rs"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+ethers = { version = "2.0", features = ["ws", "rustls", "ipc"] }
+ethers-providers = { version = "2.0", features = ["ipc"] }
```

Comment:
If possible, let's use alloy because it's the new ethers, but I don't really care for now either way